src/apic-extension/azext_apic_extension/aaz/latest/apic/integration/_create.py

```diff
@@ -11,14 +11,18 @@
 from azure.cli.core.aaz import *
 
 
+@register_command(
+    "apic integration create",
+    is_preview=True,
+)
 class Create(AAZCommand):
     """Create new or updates existing API source.
     """
 
     _aaz_info = {
-        "version": "2024-06-01-preview",
+        "version": "2024-12-01-preview",
```

Comment:
As talked in yesterday's sync meeting, before merge this PR, please manually change the API version to `2024-06-01-preview`, so people can use it in staging environment.

Reply:
I think for aaz_info, using 2024-12-01-preview is ok. We only need to change the version in `query_parameters`